### ISSUE TITLE
Redeclaration of enums causes a lot of warnings

### ISSUE THREAD

**Issue by timosachsenberg** [OpenMS/OpenMS#5327]
We probably need a way to handle redeclaration.
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:47:8: 'STRING_VALUE' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:48:8: 'INT_VALUE' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:49:8: 'DOUBLE_VALUE' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:50:8: 'STRING_LIST' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:51:8: 'INT_LIST' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:52:8: 'DOUBLE_LIST' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/DataValue.pxd:53:8: 'EMPTY_VALUE' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/ControlledVocabulary.pxd:80:8: 'NONE' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/SpectrumSettings.pxd:54:8: 'UNKNOWN' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/MassAnalyzer.pxd:167:6: 'NONE' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/TargetedExperimentHelper.pxd:16:12: 'UNKNOWN' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/TargetedExperimentHelper.pxd:29:12: 'UNKNOWN' redeclared
warning: /home/sachsenb/OMS/OpenMS/src/pyOpenMS/pxds/ReactionMonitoringTransition.pxd:74:8: 'UNKNOWN' redeclared

potential solutions? https://stackoverflow.com/questions/31001918/wrap-enum-class-for-cython

_Originally posted by @timosachsenberg in https://github.com/OpenMS/OpenMS/discussions/5275#discussioncomment-604748_

**Comment by timosachsenberg** [comment]
https://github.com/OpenMS/OpenMS/issues/4505
maybe here: https://github.com/uweschmitt/autowrap/blob/2dcf9724294f23f87ae686f7d5a5f193a5fab3e2/autowrap/CodeGenerator.py#L308

**Comment by timosachsenberg** [comment]
@jpfeuffer I think we could switch to enum class in c++ and pyopenms. But I guess this is a breaking change

**Comment by jpfeuffer** [comment]
Yes correct. Although the changes probably do not affect many users. I think the general usage will be the same. Those enums just might behave a little different when you do things with them other than initializing and passing to a function.

**Comment by timosachsenberg** [comment]
fixed
